frontend/src/components/dashboard/aliases/Alias.module.scss

```diff
@@ -171,6 +171,32 @@ $toggleTransitionDuration: 200ms;
   }
 }
 
+.block-level-label-wrapper {
+  display: none;
+  align-items: center;
+
+  @media screen and #{$mq-md} {
```

Comment:
Praise: Noticed that all the media queries in `/frontend` have `@media screen and` instead of just the `min-width` arguments. I dig the specificity.

Reply:
Heh, to be fair, that's VSCode autocompleting that for me when I type `@media` :P